tenzir/integration/tests/pipelines_local.bats

```diff
@@ -522,3 +522,13 @@ EOF
 @test "set operator" {
   check tenzir 'version | set foo="patch", :uint64=-1, :ip=1.1.1.1, qux=foo, version=123, #schema="foo.bar", schema=#schema, build=null | set schema2=#schema, baz=patch'
 }
+
+# bats test_tags=pipelines, deduplicate
+@test "Deduplicate operator" {
+  check tenzir "from ${INPUTSDIR}/json/all-types.json | deduplicate --limit 1"
+  check tenzir "from ${INPUTSDIR}/json/all-types.json | deduplicate e --limit 1"
+  check tenzir "from ${INPUTSDIR}/json/all-types.json | deduplicate b --limit 1"
+  check tenzir "from ${INPUTSDIR}/json/all-types.json | batch 1 | deduplicate b --limit 1"
+  check tenzir "from ${INPUTSDIR}/json/all-types.json | deduplicate b --limit 1 --distance 1"
+  check tenzir "from ${INPUTSDIR}/json/all-types.json | deduplicate b,e --limit 1"
+}
```

Comment:
Can we also add tests for timeouts, and can we add tests with a larger than 1 distance and/or limit, and with type extractors like `:ip` as well?

Reply:
Done.